### ISSUE TITLE
JSON-Output throws an error

### ISSUE THREAD

**Issue by tomarcade** [dmachard/dnstap-receiver#25]
Hi
When having to JSON-outputs configured (TCP and file), I got the following error:

```
$ dnstap_receiver -c /etc/dnstap/dnstap.conf -v
2021-03-04 08:43:38,043 DEBUG External config file loaded
2021-03-04 08:43:38,043 DEBUG Start receiver...
2021-03-04 08:43:38,043 DEBUG Output handler: tcp
2021-03-04 08:43:38,043 DEBUG Output handler: file
2021-03-04 08:43:38,043 DEBUG Output handler: stdout
2021-03-04 08:43:38,044 DEBUG Input handler: unix socket
2021-03-04 08:43:38,044 DEBUG Input handler: listening on /var/run/dnsdist/unix.sock
2021-03-04 08:43:38,044 DEBUG Output handler: TCP enabled
2021-03-04 08:43:38,044 DEBUG Output handler: connection to 127.0.0.1:5000
2021-03-04 08:43:38,050 DEBUG Webserver: listening on 0.0.0.0:8080
2021-03-04 08:43:38,051 DEBUG Output handler: connected
2021-03-04 09:06:51,412 DEBUG Input handler: new connection from (unix-socket)
2021-03-04 09:06:51,415 DEBUG Input handler: control ready received from (unix-socket)
2021-03-04 09:06:51,415 DEBUG Input handler: sending control accept to (unix-socket)
2021-03-04 09:06:51,416 DEBUG Input handler: control start received from (unix-socket)
Task exception was never retrieved
future: <Task finished coro=<handle() done, defined at /root/.local/lib/python3.6/site-packages/dnstap_receiver/outputs/output_file.py:34> exception=KeyError('payload',)>
Traceback (most recent call last):
  File "/root/.local/lib/python3.6/site-packages/dnstap_receiver/outputs/output_file.py", line 45, in handle
    msg = transform.convert_dnstap(fmt=output_cfg["format"], tapmsg=tapmsg)
  File "/root/.local/lib/python3.6/site-packages/dnstap_receiver/outputs/transform.py", line 36, in convert_dnstap
    del tapmsg["payload"]; del tapmsg["time-sec"]; del tapmsg["time-nsec"];
KeyError: 'payload'
```

```
...
  file:
    enable: true
    format: json
    file: /tmp/json.log
    file-max-size: 10M
    file-count: 10
  tcp-socket:
    enable: true
    format: json
    delimiter: "\n"
    retry: 5
    remote-address: 127.0.0.1
    remote-port: 5000
...
```

Switching the file-output to "text", then it works.

Many thanks.
Kind regards,
Thomas

**Comment by dmachard** [comment]
it should be fixed in the commit https://github.com/dmachard/dnstap-receiver/commit/1e3a1ad00288f75afe17c77ede2e37a1f2c3a546

**Comment by dmachard** [comment]
fix in release 4.2.1
